Source/x11/XGDragView.m

```diff
@@ -54,7 +54,7 @@
 
 #define	DRAGWINDEV [XGServer _windowWithTag: [_window windowNumber]]
 #define	XX(P)	(P.x)
-#define	XY(P)	(DisplayHeight(XDPY, DRAGWINDEV->screen) - P.y)
+#define	XY(P)	([GSCurrentServer() boundsForScreen: DRAGWINDEV->screen].size.height - P.y)
```

Comment:
Getting a field from a structure returned by an ObjC call is a dangerous operation. On one systems null may be returned instead of a valid structure. We need to check whether the the recipient is not nil. But in this specific case we are hopefully save.